src/tests/ftest/util/daos_core_base.py

```diff
@@ -57,9 +57,6 @@ def get_test_param(self, name, default=None):
     def start_server_managers(self, force=False):
```

Comment:
This whole function definition can be removed if we are no longer using scalable endpoint as all it does now is call the inherited function of the same name.

Reply:
done